### PR TITLE
Feature list

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added two types of list - normal list and stacked list

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
you can merge it

File: list/list.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  width: 50%;
+  min-width: 20rem;
+  border: 1px solid var(--dark-black);
+  background-color: var(--off-white);
+}
+.stacked-list-item {
+  padding: 1rem;
+  border-bottom: 1px solid var(--dark-black);
+  list-style-type: none;
+  display: flex;
```
Comment (tushartiwari7) [inline list/list.css:44]: use utility classes for `flex`

**Review by tushartiwari7 — APPROVED** [review]
Seems good! merge this after implementing changes.

File: list/list.css (hunk 1 above)
Comment (Bhakti199) [reply list/list.css:44]: Sure, I'm adding it at last once the components are ready, so as to avoid confusion right now.

**Pull request merged**